hedera-node/src/test/java/com/hedera/services/utils/accessors/SignedTxnAccessorTest.java

```diff
@@ -173,7 +123,7 @@ void getsCryptoSigMappingFromKnownRationalizedMeta() {
 
     @Test
     void uncheckedPropagatesIaeOnNonsense() {
-        final var nonsenseTxn = buildTransactionFrom(ByteString.copyFromUtf8("NONSENSE"));
+        final var nonsenseTxn = TxnUtils.buildTransactionFrom(ByteString.copyFromUtf8("NONSENSE"));
```

Comment:
Is name qualifying this static call really needed?  It's generating a large amount of diffs in this file unrelated to the purpose of the change of the PR.  The subsequent reformatting then makes it difficult to see if there are any substantive changes.

Reply:
Used static import.